From 4c18037dcc76c895a7eb05f91a64c0086d3d7bfb Mon Sep 17 00:00:00 2001
From: Eric Lundby <Eric.Lundby@gmail.com>
Date: Wed, 18 Feb 2026 16:51:20 -0700
Subject: [PATCH 1/1] fix-win-arm64

---
 conda/base/context.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/conda/base/context.py b/conda/base/context.py
index d1adb4f10..539d768a6 100644
--- a/conda/base/context.py
+++ b/conda/base/context.py
@@ -652,7 +652,7 @@ class Context(Configuration):
 
     @property
     def arch_name(self) -> str:
-        m = platform.machine()
+        m = platform.machine().lower()
         if m in non_x86_machines:
             return m
         else:
@@ -707,7 +707,7 @@ class Context(Configuration):
 
     @cache
     def _native_subdir(self) -> str:
-        m = platform.machine()
+        m = platform.machine().lower()
         if m in non_x86_machines:
             return f"{self.platform}-{m}"
         elif self.platform == "zos":
-- 
2.50.1 (Apple Git-155)

